packages/react/src/ReactStartTransition.js

```diff
@@ -9,12 +9,32 @@
 
 import ReactCurrentBatchConfig from './ReactCurrentBatchConfig';
 
+/**
+ *  If a certain number of fibers are updated inside startTransition then we suspect
+ *  that a user-space defined subscription is being wrapped by startTransition.
+ *  This is no good, they should instead use built-in APIs for concurrent mode to work correctly.
+ */
+
```

Comment:
Should this comment be moved down to line 25? It's not clear what this is annotating when placed here.

Reply:
Woops. I used to have a constant below the comment, but I removed it because console.warn expects a `static` string. I'll remove this comment